docs/validators/admin-guide/soroban-settings.mdx

```diff
@@ -102,25 +111,26 @@ If you didn't provide the `--signtxs` parameter to the `get-settings-upgrade-txs
 :::
 
 ```bash
-curl -G 'http://localhost:11626/tx' --data-urlencode 'blob=<LINE_2_OUTPUT>'
-curl -G 'http://localhost:11626/tx' --data-urlencode 'blob=<LINE_6_OUTPUT>'
-curl -G 'http://localhost:11626/tx' --data-urlencode 'blob=<LINE_10_OUTPUT>'
+curl -G 'http://localhost:11626/tx' --data-urlencode 'blob=<LINE_1_OUTPUT>'
+curl -G 'http://localhost:11626/tx' --data-urlencode 'blob=<LINE_3_OUTPUT>'
+curl -G 'http://localhost:11626/tx' --data-urlencode 'blob=<LINE_5_OUTPUT>'
+curl -G 'http://localhost:11626/tx' --data-urlencode 'blob=<LINE_7_OUTPUT>'
 ```
 
 ### 4. Verify Proposed Upgrades
 
-You can verify that the proposed upgrades have been set up using stellar-core's `dumpproposedsettings` command, providing the `ConfigUpgradeSetKey` XDR string from line 14 above:
+You can verify that the proposed upgrades have been set up using stellar-core's `dumpproposedsettings` command, providing the `ConfigUpgradeSetKey` XDR string from line 9 above:
 
 ```bash
-curl -G 'http://localhost:11626/dumpproposedsettings' --data-urlencode 'blob=<LINE_14_OUTPUT>'
+curl -G 'http://localhost:11626/dumpproposedsettings' --data-urlencode 'blob=<LINE_9_OUTPUT>'
 ```
 
 ### 5. Schedule the Upgrades
 
-Now you can schedule the upgrade on all the required validators using stellar-core's `upgrades` command, providing the `ConfigUpgradeSetKey` output from line 14 above and an agreed upon time in the future:
+Now you can schedule the upgrade on all the required validators using stellar-core's `upgrades` command, providing the `ConfigUpgradeSetKey` output from line 9 above and an agreed upon time in the future:
 
 ```bash
-curl -G 'http://localhost:11626/upgrades?mode=set&upgradetime=YYYY-MM-DDTHH:MM:SSZ' --data-urlencode 'configupgradesetkey=<LINE_14_OUTPUT>'
+curl -G 'http://localhost:11626/upgrades?mode=set&upgradetime=YYYY-MM-DDTHH:MM:SSZ' --data-urlencode 'configupgradesetkey=<LINE_9_OUTPUT>'
```

Comment:
How about suggesting `http-command` instead of `curl` to avoid encoding issues?

Reply:
Done